tests/slsa_analyzer/checks/test_provenance_l3_check.py

```diff
@@ -52,6 +51,18 @@ def download_asset(self, url: str, download_path: str) -> bool:
         return True
 
 
+class MockAnalyzeContext(AnalyzeContext):
```

Comment:
Is this class duplicated with the `MockAnalyzeContext` class within `conftest.py`?

Reply:
Yes, see commit [4230c5c](https://github.com/oracle/macaron/pull/305/commits/4230c5c4688b7d35c97125a7d493862d8c5e067e)